### ISSUE TITLE
can't install addons

### ISSUE THREAD

**Issue by thost96** [circuitdb/pimatic-node-red#14]
Hi,

i tried to install node-red-dashboard, but this failed multiple times: 
```
Error: Command failed: npm install --production node-red-dashboard
npm WARN deprecated isarray@2.0.1: Just use Array.isArray directly
npm ERR! tar.unpack untar error /root/.npm/component-emitter/1.2.1/package.tgz
npm ERR! tar.unpack untar error /root/.npm/debug/2.6.7/package.tgz
npm ERR! tar.unpack untar error /root/.npm/ms/2.0.0/package.tgz
npm ERR! Linux 4.4.38-v7+
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "--production" "node-red-dashboard"
npm ERR! node v4.8.3
npm ERR! npm  v2.15.11
npm ERR! path /home/pi/pimatic/node_modules/node-red-dashboard/node_modules/serve-static/node_modules/send/node_modules/debug/src/debug.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall utime

npm ERR! enoent ENOENT: no such file or directory, utime '/home/pi/pimatic/node_modules/node-red-dashboard/node_modules/serve-static/node_modules/send/node_modules/debug/src/debug.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not...
```

How can plugins be installed on pimatic-node-red?

**Comment by joshendriks** [comment]
Just go into the correct node-red directory:
`root@pimatic:~# cd /home/pimatic-app/node_modules/pimatic-node-red/node_modules/node-red
root@pimatic:/home/pimatic-app/node_modules/pimatic-node-red/node_modules/node-red# sudo npm install node-red-dashboard
`
I will leave this ticket open to see if I can make it load node-red plugins from the pimatic directory.

**Comment by joshendriks** [comment]
Fixed in version 1.0.4

**Comment by thost96** [comment]
i tried to install the addon using the gui not the cli, so the directory was choosen by node-red automatically. I try it again after an update.
